package.json

```diff
@@ -1,14 +1,15 @@
+
 {
-  "name": "bucklescript-tea",
+  "name": "rescript-tea",
   "version": "0.15.0",
   "description": "TEA for Bucklescript",
   "main": "index.js",
   "scripts": {
-    "clean": "bsb clean",
-    "clean:all": "bsb -clean-world",
-    "build": "bsb",
-    "build:all": "bsb -make-world",
-    "watch": "bsb -w",
+    "clean": "rescript clean",
```

Comment:
Can you make these build targets work?

We can also remove the pre-commit and dual-syntax stuff, since we don't have that anymore.

Reply:
rescript clean -with-deps should do the same thing as bsb -make-world